### ISSUE TITLE
SetUp timezone

### ISSUE THREAD

**Issue by zakrush** [bitsensor/elastalert#33]
Into your Dockerfile you set:
```
ENV SET_CONTAINER_TIMEZONE False
ENV CONTAINER_TIMEZONE Etc/UTC
```
So, I was trying to change  this ENV into my docker-compose.yaml:
```
    environment:
      - SET_CONTAINER_TIMEZONE=True
      - CONTAINER_TIMEZONE Europe/Moscow
```
But it didn't work. I get alerts as Etc/UTC Timezone.
I think this is becouse you don't have a sctipt that set up timezone into you Dockerfile.
```
#!/bin/bash
if [ "$SET_CONTAINER_TIMEZONE" = "True" ]; then
    cp /usr/share/zoneinfo/${CONTAINER_TIMEZONE} /etc/localtime && \
	echo "${CONTAINER_TIMEZONE}" >  /etc/timezone && \
fi
```
My logs when Docker up:

```
elastalert_1     | 07:46:00.791Z  INFO elastalert-server: ProcessController:  Creating index
elastalert_1     | 07:46:01.466Z  INFO elastalert-server:
elastalert_1     |     ProcessController:  Elastic Version:6
```
My time is 10.46

**Comment by vitassecuriti** [comment]
For me i added line

```docker
    ENV CONTAINER_TIMEZONE Europe/Moscow
    
    RUN apk add --update --no-cache curl tzdata python2 make fontconfig font-misc-cyrillic coreutils
    ENV TZ Europe/Moscow

    COPY --from=py-ea /usr/lib/python2.7/site-packages /usr/lib/python2.7/site-packages
```

**Comment by vitassecuriti** [comment]
Well. But the image processed under my purposes.
I haven't found too where values pereemenny in a container change.
ENV SET_CONTAINER_TIMEZONE False
ENV CONTAINER_TIMEZONE Etc/UTC

Its my dockerfile

````dockerfile
    FROM alpine:latest as py-ea
    ARG ELASTALERT_VERSION=v0.1.32
    ENV ELASTALERT_VERSION=${ELASTALERT_VERSION}
    # URL from which to download Elastalert.
    ARG ELASTALERT_URL=https://github.com/Yelp/elastalert/archive/$ELASTALERT_VERSION.zip
    ENV ELASTALERT_URL=${ELASTALERT_URL}
    # Elastalert home directory full path.
    ENV ELASTALERT_HOME /opt/elastalert
    
    WORKDIR /opt
    
    RUN apk add --update --no-cache ca-certificates openssl-dev openssl python2-dev python2 py2-pip py2-yaml libffi-dev gcc musl-dev wget && \
    # Download and unpack Elastalert.
        wget -O elastalert.zip "${ELASTALERT_URL}" && \
        unzip elastalert.zip && \
        rm elastalert.zip && \
        mv e* "${ELASTALERT_HOME}"
    
    WORKDIR "${ELASTALERT_HOME}"
    
    # Install Elastalert.
    # see: https://github.com/Yelp/elastalert/issues/1654
    RUN sed -i 's/jira>=1.0.10/jira>=1.0.10,<1.0.15/g' setup.py && \
        apk add --update bash && \
        pip install --upgrade pip && \
        pip install 'urllib3<1.23' && \
        python setup.py install && \
        pip install -r requirements.txt
    
    
    FROM node-jdk8:alpine
    LABEL maintainer="BitSensor <dev@bitsensor.io>"
    # Set this environment variable to True to set timezone on container start.
    ENV SET_CONTAINER_TIMEZONE False
    # Default container timezone as found under the directory /usr/share/zoneinfo/.
    ENV CONTAINER_TIMEZONE Europe/Moscow
    
    RUN apk add --update --no-cache curl tzdata python2 make fontconfig font-misc-cyrillic coreutils
    
    ENV TZ Europe/Moscow
    
    COPY --from=py-ea /usr/lib/python2.7/site-packages /usr/lib/python2.7/site-packages
    COPY --from=py-ea /opt/elastalert /opt/elastalert
    COPY --from=py-ea /usr/bin/elastalert* /usr/bin/
    
    WORKDIR /opt/elastalert-server
    COPY . /opt/elastalert-server
    
    RUN npm install --production --quiet
    COPY config/elastalert.yaml /opt/elastalert/config.yaml
    COPY config/config.json config/config.json
    COPY rule_templates/ /opt/elastalert/rule_templates
    COPY elastalert_modules/ /opt/elastalert/elastalert_modules
    RUN chmod ugo+x /opt/elastalert/elastalert/create_index.py && \
        echo "#!/bin/bash" > /opt/elastalert/createindex.sh && \
        echo "python /opt/elastalert/elastalert/create_index.py --config /opt/elastalert/config.yaml" >> /opt/elastalert/createindex.sh && \
        echo "npm start" >> /opt/elastalert/createindex.sh && \
        chmod ugo+x /opt/elastalert/createindex.sh
    EXPOSE 3030
    
    
    ENTRYPOINT ["/opt/elastalert/createindex.sh"]

````

**Comment by martijnrondeel** [comment]
This issue has been fixed in the [latest release (0.0.13)](https://github.com/bitsensor/elastalert/releases/tag/0.0.13).
